shared/xccdf/system/software/integrity.xml

```diff
@@ -310,6 +310,45 @@ Bugzilla #1275532.
 <ref nist="AC-6,AU-9(1),AU-9(3),CM-6(d),CM-6(3)" disa="1494,1496" pcidss="Req-11.5" cis="1.2.6,6.1.3,6.1.4,6.1.5,6.1.6,6.1.7,6.1.8,6.1.9,6.2.3" srg="SRG-OS-000257-GPOS-00098,SRG-OS-000278-GPOS-00108" cjis="5.10.4.1" cui="3.3.8,3.4.1" />
 </Rule>
 
+<Rule id="rpm_verify_ownership" severity="high" prodtype="rhel7">
+<title>Verify and Correct Ownership with RPM</title>
+<description>
+The RPM package management system can check file ownership
+permissions of installed software packages, including many that are
+important to system security. After locating a file with incorrect
+permissions, which can be found with
+<pre>$ rpm -Va | grep "^.....\(U\|.G\)"</pre>
+
+run the following command to determine which package owns it:
+<pre>$ rpm -qf <i>FILENAME</i></pre>
+<br />
+Next, run the following command to reset its permissions to
+the correct values:
+<pre>$ sudo rpm --setugids <i>PACKAGENAME</i></pre>
+<br />
+</description>
+<ocil clause="there is output">
+The following command will list which files on the system have ownership different from what
+is expected by the RPM database:
+<pre>$ rpm -Va --nofiledigest | awk '$1 ~ /[UG]/{print $3}'</pre>
```

Comment:
change this to `<pre>$ rpm -Va | grep "^.....\(U\|.G\)"</pre>`

Reply:
But we should return a list of files (according to PR #2205), if we use <pre>$ rpm -Va | grep "^.....\(U\|.G\)"</pre>, must add cut to parse the output and get a list of files

Reply:
@bgjoseluis no. That PR has no effect for this. This is all human-readable content and not for scripts to use or consume.

Reply:
OK thanks. Done